src/visualization/minimap.cc

```diff
@@ -355,7 +357,7 @@ void VisualizationMinimap::initTextures() {
   texture_->allocateStorage();
 
   point_size_ = std::max(1.0, static_cast<double>(sample_size_) / texture_size);
-  const uint8_t* rowdata = reinterpret_cast<const uint8_t*>(sampler_->data());
+  auto* rowdata = reinterpret_cast<const uint8_t*>(sampler_->data());
```

Comment:
const?

Reply:
`rowdata` -> `row_data`

Reply:
this is already const, no need for adding it (same for '*', but IMO it increases readability).
changed to `row_data`